### ISSUE TITLE
script:contains() doesn't seem to handle `script/rocketscript` script tags?

### ISSUE THREAD

**Issue by fake-name** [gorhill/uBlock#2354]
### Describe the issue
I'm trying to block some of the particularly spectacularly obnoxious ads, using the `script:contains()` command, and it seems to not be working?

### One or more specific URLs where the issue occurs

`https//www.nepustation.com/`  
Probably others. This is largely a function of blockadblock.com's functionality.
 
### Steps for anyone to reproduce the issue

I've created the following custom filters:

```
www.nepustation.com##script:contains(KillAdBlock)
www.nepustation.com##script:contains(kill_adBlock_status)
www.nepustation.com##script:contains(document.write)
www.nepustation.com##script:contains(Place this code snippet near the footer of your page before the close of the /body tag)
```

I'd expect this to block the blockadblock script, but it doesn't seem to work.
Either I'm misunderstanding how the `script:contains()` command works (the fact that it doesn't require "" around the script string makes me suspect I'm doing something wrong), or I don't know what's going on.

### Your settings
 
- Browser/version: Vivaldi 1.6.689.40 (Stable channel) (32-bit)
- uBlock Origin version: 1.10.4

##### Your filter lists

Defaults

##### Your custom filters (if any)

See above.

------

There's also some fun other bullshit here: The site uses a custom font with reordered codepoints to "protect" it's content from copy+paste (and break screen-readers, reading mode, and basically kill kittens for every refresh too).

**Comment by uBlock-user** [comment]
Are you on Vivaldi ? If so, inline-script tag filtering will not work. That's a feature only working on Firefox and its derivatives.

**Comment by mapx-** [comment]
however, use 
`@@||nepustation.com^$generichide`

**Comment by toshiya44** [comment]
On firefox. Disabling generichide didn't work.

**Comment by uBlock-user** [comment]
add 
````
nepustation.com##script:inject(bab-defuser.js)
@@||nepustation.com^$elemhide

**Comment by toshiya44** [comment]
Above solution didn't work either.
Apparently the site checks to see if `http://ajax.cloudflare.com/cdn-cgi/nexp/dok3v=f2befc48d1/cloudflare.min.js` is loaded. Allowing it unblocks the site.
What does this Javascript do?

**Comment by toshiya44** [comment]
nepustation.com##body:style(visibility: visible !important;)
This works

**Comment by mapx-** [comment]
the generichide (or elemhide) filter is working fine (tested i firefox / chrome + ubo / ABP)

**Comment by fake-name** [comment]
> Are you on Vivaldi ? 

Wow, I fail at reporting.

>  If so, inline-script tag filtering will not work. That's a feature only working on Firefox and its derivatives.

I fail at reading the documentation as well, apparently.  That explains it. Is  there a timeline for getting this in non-firefox browsers?

**Comment by mapx-** [comment]
read
https://github.com/gorhill/uBlock/wiki/Inline-script-tag-filtering

`Not supported in Chromium-based browser` due to
https://bugs.chromium.org/p/chromium/issues/detail?id=168175

**Comment by uBlock-user** [comment]
My solution works fine on Chromium, I haven't tested on Firefox, as I don't use Firefox, but it should. Update all your uBlock filters, and uBlock resources. 

@fake-name > Is there a timeline for getting this in non-firefox browsers?

Never Read - https://github.com/whatwg/html/issues/943

**Comment by fake-name** [comment]
@uBlock-user  - Yeah, reading that thread and feeling raeg at the moment. I love all the "people should use <other thing that's not even specified yet>" responses.

**Comment by uBlock-user** [comment]
@fake-name FYI, It might get removed from Firefox too - https://bugzilla.mozilla.org/show_bug.cgi?id=1286822

**Comment by fake-name** [comment]
It's fine if they remove it, as long as **the replacement is implemented *first***.

I'd assume they're going to remove it, and then never bother implementing any replacement, though.

**Comment by uBlock-user** [comment]
There won't be any replacement, they're following in the steps of what Google and other decided.

**Comment by gorhill** [comment]
nepustation.com##script:inject(bab-defuser.js)
    nepustation.com##script:inject(abort-on-property-write.js, KillAdBlock)

**Comment by gorhill** [comment]
Duplicate of #2015.
